tutorials/MultiQubitGates/MultiQubitGates.ipynb

```diff
@@ -481,6 +481,177 @@
+    "+ x_{100} CINOT|100\\rangle + x_{101} CINOT|101\\rangle + x_{110} CINOT|110\\rangle + x_{111} CINOT|111\\rangle \\\\\n",
+    "= x_{000}|000\\rangle + x_{001}|001\\rangle + x_{010}|010\\rangle + x_{011}|011\\rangle + \\color{red}{x_{101}}|100\\rangle + \\color{red}{x_{100}}|101\\rangle +  \\color{red}{x_{111}}|110\\rangle + \\color{red}{x_{110}}|111\\rangle $$\n",
+    "\n",
+    "**In Matrix Form**:\n",
+    "\n",
+    "$\\text{CINOT}$ can also be represented in matrix form. It will be a $2^3$ matrix\n",
+    "$$\\begin{bmatrix}\n",
+    "    1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 1 & \\color{blue} 0 & \\color{blue} 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 1 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 1 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 1 & \\color{blue} 0\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Applying $\\text{CINOT}$ to $|\\psi\\rangle$ gives us \n",
+    "$$\n",
+    "CINOT \\begin{bmatrix}\n",
+    "    1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 1 & \\color{blue} 0 & \\color{blue} 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 1 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 0 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 1 \\\\\n",
+    "    0 & 0 & 0 & 0 & \\color{blue} 0 & \\color{blue} 0 & \\color{blue} 1 & \\color{blue} 0\n",
+    "\\end{bmatrix} \n",
+    "\\begin{bmatrix} x_{000} \\\\ x_{001} \\\\ x_{010} \\\\ x_{011} \\\\ x_{100} \\\\ x_{101} \\\\ x_{110} \\\\ x_{111} \\end{bmatrix}\n",
+    "= \\begin{bmatrix} x_{000} \\\\ x_{001} \\\\ x_{010} \\\\ x_{011} \\color{green}{\\\\ x_{101} \\\\ x_{100} \\\\ x_{110} \\\\ x_{111}} \\end{bmatrix}\n",
+    "$$\n",
+    "\n",
+    "However since as $N$ gets larger, creating a full size matrix can be extremely unwieldy. Since fundamentally the gate is acting only on the 1st and 3rd qubit. We can use the $\\text{SWAP}$ gate to bring the non-adjacent qubits adjacent. \n",
+    "The steps are:\n",
+    "\n",
+    "1. Apply the $\\text{SWAP}$ gate on the 1st and 2nd qubits.\n",
+    "2. Apply the $\\text{CNOT}$ on 2nd and 3rd qubits.\n",
+    "3. Apply the $\\text{SWAP}$ gate again on the 1st and 2nd qubits.\n",
+    "\n",
+    "These can be represented as applying the following gates on the 3 qubits:\n",
+    "\n",
+    "1. $SWAP \\otimes I$\n",
```

Comment:
The fact that we implement this in Q# as `SWAP(q1,q2)` without referencing the third qubit, despite the fact that of course the gate $SWAP \\otimes I$ is applied to the whole system is a real source of confusion (https://codeforces.com/blog/entry/77614?#comment-641801 and thread below this). I think it's worth explicitly mentioning that when gates are applied to seemingly only some qubits in the system (e.g `SWAP(q1,q2)`), in fact we're applying a unitary matrix to the whole system. My suggested change on line 548 highlights this.